src/test/java/org/sagebionetworks/bridge/services/MasterSchedulerServiceTest.java

```diff
@@ -0,0 +1,202 @@
+        
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        
+        when(configDao.createSchedulerConfig(configCaptor.capture())).thenReturn(mockConfig);
+        
+        schedulerService.createSchedulerConfig(mockConfig);
+        MasterSchedulerConfig config = configCaptor.getValue();
+        
+        assertEquals(config.getScheduleId(), SCHEDULE_ID);
+        assertEquals(config.getCronSchedule(), CRON_SCHEDULE);
+        assertTrue(config.getRequestTemplate().get("a").booleanValue());
+        assertEquals(config.getRequestTemplate().get("b").textValue(), "string");
+        assertEquals(config.getSqsQueueUrl(), SQS_QUEUE_URL);
+        assertEquals(config.getVersion(), new Long(VERSION));
+        assertSame(mockConfig, config);
+        
+        verify(configDao).createSchedulerConfig(configCaptor.capture());
+    }
+    
+    @Test(expectedExceptions = InvalidEntityException.class)
+    public void testCreateSchedulerConfigValidates() {
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        mockConfig.setScheduleId(null);
+        
+        schedulerService.createSchedulerConfig(mockConfig);
+        verify(configDao, never()).createSchedulerConfig(any());
+    }
+    
+    @Test
+    public void testGetSchedulerConfig() {
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        
+        when(configDao.getSchedulerConfig(SCHEDULE_ID)).thenReturn(mockConfig);
+        
+        MasterSchedulerConfig config = schedulerService.getSchedulerConfig(SCHEDULE_ID);
+        
+        assertEquals(config.getScheduleId(), SCHEDULE_ID);
+        assertEquals(config.getCronSchedule(), CRON_SCHEDULE);
+        assertTrue(config.getRequestTemplate().get("a").booleanValue());
+        assertEquals(config.getRequestTemplate().get("b").textValue(), "string");
+        assertEquals(config.getSqsQueueUrl(), SQS_QUEUE_URL);
+        assertEquals(config.getVersion(), new Long(VERSION));
+        assertSame(mockConfig, config);
+        verify(configDao).getSchedulerConfig(SCHEDULE_ID);
+    }
+    
+    @Test(expectedExceptions = EntityNotFoundException.class)
+    public void testGetSchedulerConfigThatDoesNotExist() {
+        when(configDao.getSchedulerConfig(SCHEDULE_ID)).thenReturn(any());
+        schedulerService.getSchedulerConfig(SCHEDULE_ID);
+        
+        verify(configDao).getSchedulerConfig(SCHEDULE_ID);
+    }
+    
+    @Test
+    public void testUpdateSchedulerConfig() {
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        when(configDao.getSchedulerConfig(SCHEDULE_ID)).thenReturn(mockConfig);
+        
+        ObjectNode objNode = JsonNodeFactory.instance.objectNode();
+        objNode.put("a", true);
+        objNode.put("b", "string2");
+        
+        MasterSchedulerConfig updatedConfig = MasterSchedulerConfig.create();
+        updatedConfig.setScheduleId(SCHEDULE_ID);
```

Comment:
We should set a different schedule ID here to make sure that the update() method sets schedule ID correctly.